p4_symbolic/bmv2/test.cc

```diff
@@ -26,10 +26,10 @@
 #include "google/protobuf/util/json_util.h"
 #include "google/protobuf/text_format.h"
 
-#include "src/protobuf/bmv2/program.pb.h"
+#include "p4_symbolic/bmv2/bmv2.pb.h"
 
 namespace p4_symbolic {
```

Comment:
I don't think we want these to be in any namesapce, since this is a main executable.

Reply:
Yeah that makes sense